recipes/pennylane-lightning-kokkos/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "pennylane-lightning-kokkos" %}
+{% set version = "0.29.1" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PennyLaneAI/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f51ba7718defc7bb5064f690f381e04b2ec58cb09f22a171ae5f410860716e30
+  patches:
+    - 0001-setup.patch
+    - 0002-setup.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: |
+    {{ PYTHON }} setup.py build_ext --define="PLKOKKOS_ENABLE_WARNINGS=OFF"
+    {{ PYTHON }} setup.py bdist_wheel
+    {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+    - kokkos =3.7
+    - pip
+    - pybind11
+    - python
+  run:
+    - pennylane
```

```suggestion
    - pennylane >=0.28
```
https://github.com/PennyLaneAI/pennylane-lightning-kokkos/blob/88d5657159a6fb177b6c4dfec66963201bff47ad/setup.py#LL143C20-L143C20